bookmarks/images/views.py

```diff
@@ -29,3 +31,18 @@ def image_detail(request, id, slug):
     return render(
         request, "images/image/detail.html", {"section": "images", "image": image}
     )
+
+
+@login_required
+@require_POST
+def image_like(request):
```

Comment:
```suggestion
def like_image_view(request):
```
image_like name projects itself as something like an image. The action(verb) here is like i.e. the view is responsible for adding user's like to an image.